docs/development/integration_tests.md

```diff
@@ -31,8 +31,9 @@ If the Control Cluster is a Gardener Shoot cluster then,
 
 If the Control Cluster is a Gardener SEED cluster, then the suite ideally employs the already existing `MachineClass` and Secrets. However,
 
-1. Define the variable `IS_CONTROL_SEED` in the `.env` file and set it to `true`. 
+1. Define the variable `IS_CONTROL_CLUSTER_SEED` in the `.env` file and set it to `true`. 
 `Warning:` Make sure to set the `CONTROL_NAMESPACE` variable to the shoot namespace where the control plane of the target resides.
+1. Please pass `TARGET_RESOURCE_GROUP` in the `.env` file and set it to the name of the target cluster. This is only required for Azure clusters.
```

Comment:
Is this env var mandatory for azure clusters? if so maybe need to be a bit more clear

Reply:
yes this is mandatory for Azure